src/nunit.gui.core/Model/Settings/GuiSettings.cs

```diff
@@ -59,18 +59,11 @@ public class MainFormSettings : SettingsWrapper
     {
         public MainFormSettings(ISettings settingsService) : base(settingsService, "Gui.MainForm") { }
 
-        private const string locationKey = "Location";
-        public Point Location
+        private const string windowShapeKey = "LocationAndSize";
+        public WindowShape LocationAndSize
```

Comment:
To be honest, I'm not crazy about this idea. Location and Size are concepts in most widget platforms I'm familiar with. I haven't seen a WindowShape before. Is this something from WPF?